packages/common/package.json

```
@@ -21,7 +21,8 @@
     "start": "rollup -c --bundleConfigAsCjs --watch",
     "lint": "eslint . && stylelint \"src/**/*.css\" --allow-empty-input",
     "lint:fix": "eslint . --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
-    "test": "TZ=UTC jest",
+    "test": "TZ=UTC jest && npm run test:TZ",
```

```suggestion
    "test": "TZ=UTC jest",
```

none UTC is not a use case, we can leave the `test:TZ` as an optional test

---

It's user's time zone (used by the browser when executing the code).

---

great, so can you change as requested ?

---

ok, makes sense :+1: